### PR TITLE
screencap: add Turkish translation

### PR DESCRIPTION
**Pull request by seppacar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.tr/android/screencap.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# screencap
+
+> Bir mobil ekranın ekran görüntüsünü alın.
+> Bu komut sadece `adb shell` üzerinden kullanılabilir.
+> Daha fazla bilgi için: <https://developer.android.com/studio/command-line/adb#screencap>.
```
Comment (kbdharun) [inline pages.tr/android/screencap.md:5]:
```suggestion
> Daha fazla bilgi: <https://developer.android.com/studio/command-line/adb#screencap>.
```

File: pages.tr/android/screencap.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+- Bir ekran görüntüsü al:
+
+`screencap {{path/to/file}}`
```
Comment (kbdharun) [inline pages.tr/android/screencap.md:9]:
```suggestion
`screencap {{dosya/yolu}}`
```
Translate placeholders/command arguments for better readability to native readers.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**